### DISCUSSION TITLE
How do I set the default selection for QuickPickItem

### DISCUSSION THREAD

**Discussion by MrZmcH** [eclipse-theia/theia#13175] [General]
for example 
`{picked: true}`

**Comment by msujew** [comment]
@MrZmcH You need to set the `activeItems` on the quick pick itself. See:

https://github.com/eclipse-theia/theia/blob/ad2e10691147fa0184fc96cd17d213f59669ca07/packages/core/src/browser/i18n/language-quick-pick-service.ts#L35-L49

**Reply by MrZmcH** [reply]
Thank you,I have already used QuickPickService,  So,  is this only supported by QuickInputService?

**Reply by msujew** [reply]
You should be able to pass `{ activeItem: selected }` to the options of the `QuickPickService#show` method.

**Reply by MrZmcH** [reply]
QuickPickItem And QuickPickOptions , Don't seem to support either

**Reply by msujew** [reply]
`QuickPickOptions` definitely supports it:

https://github.com/eclipse-theia/theia/blob/0910b027f9606e676a0a73055c88fdadd20176fc/packages/core/src/common/quick-pick-service.ts#L261

**Reply by MrZmcH** [reply]
Thank you，Thank you for your answer, this may be a version issue, my theia is using version 1.14.
![image](https://github.com/eclipse-theia/theia/assets/45864434/87977ea3-6d5d-4180-a182-39ab2430932e)

**Reply by msujew** [reply]
@MrZmcH I see. It is most certainly a version issue. That version is now 3 years old, you should consider upgrading.

**Reply by MrZmcH** [reply]
now,The latest version which is the most stable?

**Reply by msujew** [reply]
Generally, we always recommend either the latest release (1.44.0) or the current community release (1.43.1).

**Reply by MrZmcH** [reply]
I did not find the version number of each version of theia corresponding to the dependency, such as yarn, node, python, is there such a comparison table?

**Reply by msujew** [reply]
We always use yarn v1, as for everything else: We only show what's currently needed. See [here](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites). We don't expect users to upgrade from one outdated version to another outdated version.

**Reply by MrZmcH** [reply]
Thank you,I will try @msujew
